### ISSUE TITLE
dev-corpus2 breaks preText

### ISSUE THREAD

**Issue by kbenoit** [quanteda/quanteda#1586]
From revdepcheck

# preText

Version: 0.6.2

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
                          Arthur Spirling, NYU
      Type vignette('getting_started_with_preText') to get started.
      Development website: https://github.com/matthewjdenny/preText
      > 
      > test_check("preText")
      [31m──[39m [31m1. Error: Small example works (@test_factorial_preprocessing.R#6) [39m [31m───────────────────────────────────[39m
      unused argument (alist())
      1: documents[1:10, ] at testthat/test_factorial_preprocessing.R:6
      
      ══ testthat results  ════════════════════════════════════════════════════════════════════════════════════
      OK: 0 SKIPPED: 0 FAILED: 1
      1. Error: Small example works (@test_factorial_preprocessing.R#6) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 60-69 (getting_started_with_preText.Rmd) 
    Error: processing vignette 'getting_started_with_preText.Rmd' failed with diagnostics:
    unused argument (alist())
    Execution halted
    ```

**Comment by koheiw** [comment]
The test should use `head.corpus()`:
https://github.com/matthewjdenny/preText/blob/486a2722d6828fa3fe5b83cc05cddb397b516aa1/tests/testthat/test_factorial_preprocessing.R#L4-L6

**Comment by kbenoit** [comment]
Broken:

*  indexing of a corpus via `data_corpus_inaugural[1:10,]`, in 
    - https://github.com/matthewjdenny/preText/blob/master/tests/t:estthat/test_factorial_preprocessing.R#L6
    - https://github.com/matthewjdenny/preText/blob/master/vignettes/getting_started_with_preText.Rmd#L66

* This accesses internals of a corpus, instead of using `ndoc()` 😡 
https://github.com/matthewjdenny/preText/blob/master/R/factorial_preprocessing.R#L80

I created a PR https://github.com/matthewjdenny/preText/pull/8 from my fork that fixes these, both for v1.4 and for branch `dev-corpus2`.

**Comment by kbenoit** [comment]
Fixed in https://github.com/matthewjdenny/preText/commit/0a8d0061407bd729e9a3eb3559fb0b3895cf340b
